.github/workflows/build_and_test.yml

```diff
@@ -71,17 +73,88 @@ jobs:
+             chmod +x bazel-4.2.3-installer-linux-x86_64.sh
+             ./bazel-4.2.3-installer-linux-x86_64.sh --user
+             export PATH="$PATH:$HOME/bin"
+             cd ${GITHUB_WORKSPACE}/ipdk.recipe/stratum/stratum
+             export DPDK_INSTALL=${GITHUB_WORKSPACE}/install
+             echo "DPDK" > ${GITHUB_WORKSPACE}/install/share/TARGET
+             xargs -a ${GITHUB_WORKSPACE}/ipdk.recipe/.github/dpdk-tests.txt bazel test --define target=dpdk
+             cd bazel-testlogs/stratum
+             zip stratum_ut_logs.zip `find . -name test.xml -print`
+                          
+      - name: 'Upload stratum-ut-result artifact'
+        uses: actions/upload-artifact@v3
+        with:
+          name: stratum-ut-result
+          path: ipdk.recipe/stratum/stratum/bazel-testlogs/stratum/stratum_ut_logs.zip
+          
       - name: 'Execute dpdk target build'
         run: |
              cd ${GITHUB_WORKSPACE}/ipdk.recipe
              export IPDK_RECIPE=${GITHUB_WORKSPACE}/ipdk.recipe
              export SDE_INSTALL=${GITHUB_WORKSPACE}/install
              export DEPEND_INSTALL=${GITHUB_WORKSPACE}/deps
              source ./scripts/dpdk/setup_env.sh $IPDK_RECIPE $SDE_INSTALL $DEPEND_INSTALL
-             ./make-all.sh --target=dpdk
+             ./make-all.sh --target=dpdk  --prefix=${GITHUB_WORKSPACE}/deps 
+          
+  run_ovs_unittests:
+    timeout-minutes: 30
+    runs-on: ubuntu-20.04
+    steps:
+      - name: 'Checkout p4-dpdk repo'
+        uses: actions/checkout@v3
+        with:
+          repository: p4lang/p4-dpdk-target
+          submodules: 'recursive'
+          path: p4sde
+
+      - name: 'Install dependencies'
+        working-directory: ./p4sde/tools/setup
+        run: |
+              sudo apt update -y
+              sudo apt install python python3-setuptools python3-pip python3-wheel python3-cffi sudo libssl-dev libatomic1 libnl-route-3-dev curl -y
+              pip3 install distro
+              python3 install_dep.py
+              cmake --version
+              sudo apt-get purge cmake
+              sudo rm -rf /usr/local/bin/cmake || true
+              sudo rm -rf ~/.local/bin/cmake || true
+              # removing cmake 3.25 and reinstalling 3.20, as build is failing with cmake 3.25
+              wget https://cmake.org/files/v3.20/cmake-3.20.2-linux-x86_64.sh -O ${GITHUB_WORKSPACE}/cmake-3.20.2-linux-x86_64.sh
+              chmod a+x ${GITHUB_WORKSPACE}/cmake-3.20.2-linux-x86_64.sh
+              sudo mkdir -p /opt/rh/cmake
+              cd ${GITHUB_WORKSPACE} && sudo sh cmake-3.20.2-linux-x86_64.sh --prefix=/opt/rh/cmake --skip-license
+              sudo ln -s /opt/rh/cmake/bin/cmake /usr/local/bin/cmake
+              cmake --version
 
+      - name: 'Checkout Networking Recipe'
+        uses: actions/checkout@v3
+        with:
+          submodules: 'recursive'
+          path: ipdk.recipe
+          
+      - name: 'Run OVS unit tests'
```

Comment:
The OVS unit tests do not need to be run on every commit. OVS is a third-party package that rarely changes. If we're going to spend the time, there are other checks that are far more valuable than this.

Reply:
Removed OVS unit test from this flow - will add individual action based on schedule/on demand builds